packages/addons-website/rollup.config.js

```diff
@@ -70,6 +71,7 @@ module.exports = [
         react: 'React',
         'react-dom': 'ReactDOM',
         'prop-types': 'PropTypes',
+        'react-scroll-up': 'ScrollToTop',
```

Comment:
Ahhh, that's what I missed

Reply:
ah me too! I tried adding this a while ago and couldn't get it working.

Reply:
It took a lot of googling error messages to figure it out 😂